<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpThreadClientClass</class>
 <widget class="QMainWindow" name="TcpThreadClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TcpThreadClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ip:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ip">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="port">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chooseFile_2">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>连接</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="filePath">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chooseFile">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>选择文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QWidget" name="widget_3" native="true">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>194</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="sendFile">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>发送文件</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>193</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TcpThreadClient.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>chooseFile_2</sender>
   <signal>clicked()</signal>
   <receiver>TcpThreadClientClass</receiver>
   <slot>connectServer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chooseFile</sender>
   <signal>clicked()</signal>
   <receiver>TcpThreadClientClass</receiver>
   <slot>chooseFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendFile</sender>
   <signal>clicked()</signal>
   <receiver>TcpThreadClientClass</receiver>
   <slot>sendFiler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>chooseFile()</slot>
  <slot>sendFiler()</slot>
  <slot>connectServer()</slot>
 </slots>
</ui>
